src/features/Instructors/InstructorsFilters/index.jsx

```diff
@@ -20,18 +24,22 @@ const initialFilterState = {
   instructorEmail: '',
   courseSelected: null,
   inputType: 'name',
+  active: false,
 };
 
 const InstructorsFilters = ({ resetPagination, isAssignSection }) => {
   const dispatch = useDispatch();
+  const debounceRef = useRef(null);
+  const { SHOW_INSTRUCTOR_FEATURES = false } = getConfig() || {};
   const selectedInstitution = useSelector((state) => state.main.selectedInstitution);
   const courses = useSelector((state) => state.courses.selectOptions);
 
   const [courseOptions, setCourseOptions] = useState([]);
 
   const { formState, handleInputChange, resetFormState } = useFormInput(initialFilterState);
 
-  const isButtonDisabled = formState.instructorEmail === '' && formState.instructorName === '' && formState.courseSelected === null;
+  const isButtonDisabled = isAssignSection && formState.instructorEmail === '' && formState.instructorName === '' && formState.courseSelected === null;
```

Comment:
Why did you added the isAssignSection to the conditions for disabled the button?

Reply:
Due to the initial approach I was taking, I had to leave the buttons unblocked, which allowed thousands of requests to be sent per second. the `isAssignSection` was used to avoid this behavior in the screen to assign instructors, but now nothing of that is necessary.